istioctl/pkg/waypoint/waypoint.go

```diff
@@ -201,6 +229,12 @@ func Cmd(ctx cli.Context) *cobra.Command {
 		fmt.Sprintf("Specify the traffic type %s for the waypoint", validTrafficTypes.String()),
 	)
 
+	waypointApplyCmd.PersistentFlags().StringVar(&enrolledNamespace,
```

Comment:
I think we should only handle "enroll the same namespace as the waypoint" probably. If they are off the typical path they can just manually do it